src/screens/PeopleList/screen/PeopleList.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {FlashList} from '@shopify/flash-list';
+import getGlobalStyles from '~/helpers/GlobalStyles';
+import getStyles from './PeopleList.styles';
+import PeopleItem from '~/components/PeopleItem';
+
+const renderItem = ({item}) => {
+  let avatarSource;
+  const {avatar, first_name, last_name, quote, address, gender} = item;
+  if (avatar) {
+    avatarSource = {uri: avatar};
+  }
+  return (
+    <PeopleItem
+      avatar={avatarSource}
+      firstName={first_name}
+      lastName={last_name}
+      gender={gender}
+      quote={quote}
+      address={address}
+    />
+  );
+};
+
+const PeopleList = () => {
+  const styles = getStyles();
+  const generalStyles = getGlobalStyles();
+  const data = require('~/assets/data/MOCK_DATA.json');
+
+  return (
+    <FlashList
+      data={data}
```

Comment:
Please move the mock data in the container and have the list render whatever data it receives as props. Also forward callback to the container to be handled there.